.gitignore

```diff
@@ -1 +1,6 @@
 venv
+redis*
+poppunk/GPS_v4
+requirements.txt
```

Comment:
I think requirements file usually left around as a tool for others to install the dependencies of your package

Reply:
https://www.idkrtm.com/what-is-the-python-requirements-txt/